zookeeper-server/src/test/java/org/apache/zookeeper/server/quorum/MultipleAddressesTest.java

```diff
@@ -110,7 +110,7 @@ public void testGetValidAddress() throws NoRouteToHostException {
     @Test
     public void testGetValidAddressWithNotValid() {
         assertThrows(NoRouteToHostException.class, () -> {
-            MultipleAddresses multipleAddresses = new MultipleAddresses(new InetSocketAddress("10.0.0.1", 22));
+            MultipleAddresses multipleAddresses = new MultipleAddresses(new InetSocketAddress("240.0.0.0", 22));
```

Comment:
Several blocks are reserved specifically for documentation/examples, and should never be routable, according to [RFC 5737](https://tools.ietf.org/html/rfc5737). I would pick from one of those, rather than one that might be reserved for some other, routable, purpose in future: `203.0.113.1` should work.
```suggestion
            // IP chosen because it is reserved for documentation/examples and should be unreachable (RFC 5737)
            MultipleAddresses multipleAddresses = new MultipleAddresses(new InetSocketAddress("203.0.113.1", 22));
```

Reply:
I agree this is a better choice, so applied the suggestion. Love the github feature of commit in place (and thanks for the suggestion!)